org.openmw.OpenMW.yaml

```diff
@@ -141,6 +141,8 @@ modules:
       - type: git
         url: https://gitlab.com/OpenMW/openmw.git
         tag: openmw-0.46.0
+      - type: patch
+        path: openmw.patch
```

Comment:
This file definitely needs a better name, something more descriptive - like `openmw-46-platform-fix`